CHANGELOG.md

```diff
@@ -2,6 +2,28 @@
 
 This changelog covers all three packages, as they are (for now) updated as a whole
 
+## v0.35.0
+
+### @tomic/react
+
+#### Breaking changes
+
+- Remove `initAgentFromLocalStorage()`.
+- No longer save agent to local storage.
+
+### @tomic/lib
+
+- Add the ability to change the `fetch` function used to fetch resources over http.
+- `store.addResource` is depricated in favor of `store.addResources`.
+- Add `AgentChange` event on store that is fired whenever the stores agent changes.
+
+#### Breaking Changes:
+
+- `uploadFiles()` has moved to `store.uploadFiles()`.
+- Remove `Agent.fromJSON()`
+- `tryValidURL` and `isValidURL` are now static methods on `Client`.
```

Comment:
Does the Client provide dependencies for these functions, e.g. is URL polyfilled when missing or something? Otherwise I don't know why it should be a static method

Reply:
They shouldn't be used outside a context where Client isn't also (indirectly) used as they check if a string can be used as subject. I think renaming them to `isValidSubject` etc. would be more appropriate since a subject being a URL is an implementation detail. Checking if something is a URL is not the concern of @tomic/lib, checking if something can be used as a subject is.

Reply:
Agreed